src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader-types.ts

```diff
@@ -1,6 +1,6 @@
 export interface McFusUploader {
-  Start(file: McFusFile): void;
-  Cancel(): void;
+  start(file: McFusFile): void;
+  cancel(): void;
 }
 
 export enum McFusMessageLevel {
```

Comment:
Not a big deal but field names also start with lowercase, usually. This looks like C# :)

Reply:
Fixed :)